test/integration/render-tests/collator/default/style.json

```diff
@@ -0,0 +1,225 @@
+              "a": "a",
+              "b": "Â",
+              "operator": "==",
+              "case-sensitive": false,
+              "diacritic-sensitive": false
+            },
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                -1.0546875,
+                30.14512718337613
+              ]
+            }
+          },
+          {
+            "type": "Feature",
+            "properties": {
+                "a": "a",
+                "b": "Â",
+                "operator": "!=",
+                "case-sensitive": false,
+                "diacritic-sensitive": false
+            },
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                -5.625,
+                71.85622888185527
+              ]
+            }
+          },
+          {
+            "type": "Feature",
+            "properties": {
+                "a": "a",
+                "b": "Â",
+                "operator": "<=",
+                "case-sensitive": true,
+                "diacritic-sensitive": false
+            },
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                1.40625,
+                -40.44694705960048
+              ]
+            }
+          },
+          {
+            "type": "Feature",
+            "properties": {
+                "a": "a",
+                "b": "ä",
+                "operator": "<",
+                "case-sensitive": true,
+                "diacritic-sensitive": false
+            },
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                -106.5234375,
+                72.18180355624855
+              ]
+            }
+          },
+          {
+            "type": "Feature",
+            "properties": {
+                "a": "a",
+                "b": "ä",
+                "operator": ">",
+                "case-sensitive": true,
+                "diacritic-sensitive": true
+            },
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                -113.5546875,
+                32.54681317351514
+              ]
+            }
+          },
+          {
+            "type": "Feature",
+            "properties": {
+                "a": "a",
+                "b": "ä",
+                "operator": ">=",
+                "case-sensitive": true,
+                "diacritic-sensitive": true
+            },
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                -114.60937499999999,
+                -43.83452678223682
+              ]
+            }
+          },
+          {
+            "type": "Feature",
+            "properties": {
+                "a": "a",
+                "b": "ä",
+                "operator": "<",
+                "case-sensitive": true,
+                "diacritic-sensitive": false
+            },
```

Comment:
This combination looks like a duplicate of the one at line 67

Reply:
Oops, meant that to be diacritic-sensitive!